modules/react/button/lib/parts/ButtonLabelDataNew.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import {styled} from '@workday/canvas-kit-react/common';
+
+export const buttonLabelDataClassName = 'button-label-data';
```

Comment:
What does this do? I don't see any reference to it in our codebase. If we find out, there should be a comment if it is important.

Reply:
this was copy over, my guess is some teams where using it but maybe i should ask Alex for more context

Reply:
Did a quick search and it looks like this class is used for some styling, we can remove it but it will be another breaking change